openedx/adg/admin_dashboard/performance/tests/factories.py

```diff
@@ -0,0 +1,16 @@
+"""
+Model factories for `performance` app
+"""
+import factory
+
+from lms.djangoapps.grades.models import PersistentCourseGrade
+
+
+class PersistentCourseGradeFactory(factory.DjangoModelFactory):
+    """
+    Factory for `PersistentCourseGrade` model. `PersistentCourseGrade` is a core model
+    and does not have a factory so adding it here as it is being used in current app.
```

Comment:
We do not need add 

> `PersistentCourseGrade` is a core model and does not have a factory so adding it here as it is being used in current app.`

 in the docstring. If you think it is important to mention, we can either comment it or add a Note